substrate/primitives/runtime/src/traits.rs

```diff
@@ -1149,6 +1149,41 @@ pub trait IsMember<MemberId> {
 	fn is_member(member_id: &MemberId) -> bool;
 }
 
+/// Super trait with all the attributes for a block number.
+pub trait BlockNumber: Member
++ MaybeSerializeDeserialize
++ MaybeFromStr
++ Debug
++ sp_std::hash::Hash
++ Copy
++ MaybeDisplay
++ AtLeast32BitUnsigned
++ Into<U256>
++ Into<u64>
```

Comment:
where is this coming from?

Reply:
Come from subxt

Reply:
What is very embarrassing is that the `bridge` pallet test relies on the `u128` type block number, which makes it impossible to add `Into<u64>`

Reply:
subxt is a rust sdk. it makes no sense to update Substrate because one of the client wants something. and yeah we shouldn't limit the block number type to be only u64.

Reply:
so bigger than u64 will be useful？